root-drive-science-ios/LogMessage+CoreDataProperties.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  LogMessage+CoreDataProperties.swift
+//  root-drive-science-ios
+//
+//  Created by Jesse Cooper on 9/24/20.
+//  Copyright © 2020 Root. All rights reserved.
+//
+//
+
+import CoreData
+import Foundation
+
+extension LogMessage {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<LogMessage> {
+        return NSFetchRequest<LogMessage>(entityName: "LogMessage")
+    }
+
+    @NSManaged public var date: Date?
```

Comment:
Are we not storing these as full timestamps, or is this just swift not being clear?

Reply:
These do include time, so in the app logs they'll show up as e.g. `[2020-09-25 16:30]`